src/main/java/frc/robot/subsystems/Rotors.java

```diff
@@ -28,9 +29,19 @@ public void setFunnelBeltMotorSpeed(double speed) {
     funnelBeltMotor.set(speed);
   }
 
+  public void setFlywheelMotorSpeed(double speed) {
+    flywheelMotor.set(speed);
+
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
   }
 
+  public void flywheelSpeed(double flywheelSpeed) {
+    // TODO Auto-generated method stub
+    throw new UnsupportedOperationException("Unimplemented method 'flywheelSpeed'");
+  }
```

Comment:
you already have something called setFlywheelMotorSpeed that is also doing something internally. Lets use that one and delete this